storage/pebble/registers.go

```diff
@@ -42,7 +72,8 @@ func (s *Registers) Get(
 	encoded := newLookupKey(height, reg).Bytes()
 	ok := iter.SeekPrefixGE(encoded)
 	if !ok {
-		return []byte{}, nil
+		// no such register found
+		return nil, storage.ErrNotFound
```

Comment:
Same to all other errors

Reply:
Should the caller wrap it then? why I said this is because when I was testing I got this error and all I could see was register not found, I had no idea which one. What is the design logic on who wraps and adds more context the first in chain that has it or last one?

Reply:
yea, exactly. The caller tried to look up a specific register at some height, but it wasn't found. If it's useful to the caller's logic, it can wrap the error including whatever context it needs. 

If we wrap it here, it's very likely that callers would also wrap the same context bloating the messages.